### DISCUSSION TITLE
You may have this issue on the last question too...

### DISCUSSION THREAD

**Discussion by millie-spencer** [earthlab-education/Earth-Analytics-2023-01-Intro#208] [Q&A02 Open Reproducible Science] [answered]
I was having an issue on the LA temperature assignment where the last question would give me an error and wouldn't produce the LA temperature plot. I talked to Elsa and it sounds like some other people have had this issue -- if you do, I recommend 1) making sure you didn't accidentally start another codespace (that's what I did, and when I went back to silver pancake the plot worked), and 2) if that doesn't work, commit and sync your changes, and then start a new codespace and try to run the workbook there. Hopefully this helps!

**Comment by eculler** [comment, marked as answer]
I recommend 1) making sure you didn't accidentally start another codespace (that's what I did, and when I went back to silver pancake the plot worked), and 2) if that doesn't work, **commit and sync your changes**, and then start a new codespace and try to run the workbook there. Hopefully this helps!

Thanks @millie-spencer !
